conf.h

```diff
@@ -96,12 +96,26 @@ static inline int __bsr32(unsigned x) { unsigned long z=0; _BitScanReverse(&z, x
 static inline int bsr32(  unsigned x) { unsigned long z;   _BitScanReverse(&z, x); return x?z+1:0; }
 static inline int ctz32(  unsigned x) { unsigned long z;   _BitScanForward(&z, x); return x?z:32; }
 static inline int clz32(  unsigned x) { unsigned long z;   _BitScanReverse(&z, x); return x?31-z:32; }
-    #ifdef _WIN64
-#pragma intrinsic(_BitScanReverse)
+    #if !defined(_M_ARM64) && !defined(_M_X64)
+static inline unsigned char _BitScanForward64(unsigned long* ret, uint64_t x) {
+  unsigned long x0 = (unsigned long)x, top, bottom;
+  _BitScanForward(&top, (unsigned long)(x >> 32));
+  _BitScanForward(&bottom, x0);
+  *ret = x0 ? bottom : 32 + top;
+  return x != 0;
```

Comment:
that dummy return is the way it is with `_BitScanForward64`/`_BitScanReverse64`. It will be optimized away anyways